test/sdk_test.dart

```diff
@@ -127,10 +127,8 @@ void main() {
         await pubCommand(
           command,
           environment: {'FLUTTER_ROOT': p.join(d.sandbox, 'flutter')},
-          error: equalsIgnoringWhitespace("""
-              Because myapp depends on foo ^1.0.0 from sdk which doesn't match
-                any versions, version solving failed.
-            """),
+          error: contains('''
+Because myapp depends on foo ^1.0.0 from sdk which doesn't match any versions, version solving failed.'''),
```

Comment:
Why these changes, what is wrong with `equalsIgnoringWhitespace` isn't that more robust?

Reply:
Well - now these output more, so we are only recognizing a substring.

We should really have a `containsIgnoringWhitespace`...

Let's create that another day.